content/en/flux/monitoring/logs.md

```diff
@@ -5,11 +5,41 @@ description: "How to monitor the Flux logs with Loki and Grafana"
 weight: 3
 ---
 
-The Flux controllers follow the Kubernetes structured logging conventions.
+The Flux controllers follow the Kubernetes structured logging conventions. These
+logs can be collected and analyzed to monitor the operations of the controllers.
+
+The [fluxcd/flux2-monitoring-example][monitoring-example-repo] repository
+provides a ready-made example setup to get started with monitoring Flux, which
+includes [Loki-stack][loki-stack] to collect logs from all the Flux controllers
+and explore them using Grafana. It is recommended to set up the monitoring
+example before continuing with this document to follow along. Before getting
+into Loki and Grafana setup, the following sections will describe the Flux logs
+and how to interpret them.
+
+## Controller logs
+
+The default installation of Flux controllers write logs to `stderr` in JSON
+format at the `info` log level. This can be configured using the
+`--log-encoding` and `--log-level` flags in the controllers. Refer to the
+[flux-system
+kustomization](https://github.com/fluxcd/flux2-monitoring-example/blob/main/clusters/test/flux-system/kustomization.yaml)
+for an example of how to patch the Flux controllers with flags. The following
+example patch snippet can be appended to the existing set of patches to add a
+log level flag and change the log level of the controller to `debug`.
+
+```yaml
+- op: add
+  path: /spec/template/spec/containers/0/args/-
```

Comment:
This will result in a crash loop of all controllers, the `--log-level` is already set at index 2.

Reply:
Oh maybe pflag changed since I last tried this. OK